operatorconfig/driverconfig/powerflex/v2.8.0/node.yaml

```diff
@@ -109,11 +109,6 @@ spec:
               value: <X_CSI_RENAME_SDC_PREFIX>
             - name: X_CSI_MAX_VOLUMES_PER_NODE
               value: <X_CSI_MAX_VOLUMES_PER_NODE>
-            - name: X_CSI_POWERFLEX_KUBE_NODE_NAME
-              valueFrom:
-                fieldRef:
-                  apiVersion: v1
-                  fieldPath: spec.nodeName
```

Comment:
These changes should be reverted. As it was a fix raised in [PR](https://github.com/dell/csm-operator/pull/426/files). Please follow the same for other node.yaml files for powerflex (v2.7.0, v2.9.0 and testfiles)

Reply:
@panigs7 will do I will revert these now